extendables/player.js

```diff
@@ -0,0 +1,13 @@
+const { Extendable } = require("klasa");
```

Comment:
1st) this shouldn't be attached to guild
2nd) if we needed this it should be a Structures.extend

Reply:
After furter discussion this was removed but music is an extendable